src/styles/components/widgets/_multi-format-text-editor.scss

```diff
@@ -212,12 +284,20 @@ $multi-format-editor-background: #fafbfd;
 }
 
 .link-editor {
-  background-color: #dcdddd;
-  border: 1px solid #ccc;
-  padding: 10px 5px;
+  background-color: $multi-format-editor-background;
+  border: $border;
+  padding: $half-spacing-unit;
   position: absolute;
-  width: 235px;
+  //width: 268px;
+  //height: 115px;
+  //left: auto !important;
+  //top: 195px !important;
+  //transform: translateX(-1px);
   z-index: 1;
```

Comment:
delete?

Reply:
@aulneau yep in general we delete rather than comment-out. The code will be in git history anyway. Otherwise they accumulate over time and it's hard to know why it was commented out.